common-service-core/src/main/java/com/ca/mfaas/monitoring/LatencyUtilsConfigInitializer.java

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
I would suggest to rather use gateway-common.

Reply:
@JirkaAichler, I suggested to put this class in common-service-core instead of gateway-common, because of this class common for every application (service), and it has not any connection with gateway.

Reply:
@JirkaAichler are you ok with common-service-core?

Reply:
Yes, @arxioly is right. I wasn't aware that it is happening for all Spring boot apps ... I am just surprised that not so many people are complaining.

Reply:
Sorry guys, Who can explain to me why apiml-common is a bad place for this?
common-service-core module just contain 'spring framework' classes. If you considered that module classes, you will see easily this module just contains core Java classes, it does not depend on any framework. Your new service may not be developed in spring. 
gateway-common module is a temporary name. There's not any connection with the gateway. (it's apiml-common)

Reply:
That's a good argument, really. But I have a one question, if this gateway-common is a temporary name, why we would not changed it? I mean, now we have already version 1.1.5, so everything temporary should already be a permanent. And this is not so big change, what is the reason not to do this in this task, for example?